Modules/Retailer/Http/Controllers/RetailerController.php

```diff
@@ -63,5 +64,256 @@ public function register(Request $request)
+        $retailer->last_name = $user->last_name;
+        $retailer->email = $user->email;
+        $retailer->verified = $user->verified;
+
+        $response = ['res' => true, 'msg' => "", 'data' => $retailer];
+        return response()->json($response);
+    }
+
+    public function updateAccount(Request $request)
+    {
+        $validator = Validator::make($request->all(), [
+            'first_name' => 'required',
+            'last_name' => 'required',
+            'phone_number' => 'required',
+        ]);
+        if ($validator->fails()) {
+            $response = ['res' => false, 'msg' => $validator->errors()->first(), 'data' => ""];
+        } else {
+            $user = User::find($request->user_id);
+            $user->first_name = $request->first_name;
+            $user->last_name = $request->last_name;
+            $retailer = Retailer::where('user_id', $request->user_id)->first();
+            $retailer->country_code = $request->country_code;
+            $retailer->country = $request->country;
+            $retailer->phone_number = $request->phone_number;
+            $retailer->language = $request->language;
+            $retailer->store_name = $request->store_name;
+            $retailer->store_type = $request->store_type;
+            $retailer->sign_up_for_email = $request->sign_up_for_email;
+            $retailer->website_url = $request->website_url;
+            $retailer->save();
+
+            $status = $user->save();
+            if ($status) {
+                if ($request->new_password != '') {
+                    $validator2 = Validator::make($request->all(), [
+                        'old_password' => 'required',
+                        'new_password' => 'required|min:6|different:old_password',
+                        'confirm_password' => 'required|same:new_password'
+                    ]);
+                    if ($validator2->fails()) {
+                        $response = ['res' => false, 'msg' => $validator2->errors()->first(), 'data' => ""];
+                    } else {
+                        if (Hash::check($request->old_password, $user->password)) {
+                            $user->password = Hash::make($request->new_password);
+                            $user->save();
+                            $response = ['res' => true, 'msg' => "Successfully updated your account", 'data' => ''];
+                        } else {
+                            $response = ['res' => false, 'msg' => 'old password does not match our record.', 'data' => ""];
+                        }
+                    }
+                } else {
+                    $response = ['res' => true, 'msg' => "Successfully updated your account", 'data' => ''];
+                }
+            } else {
+                $response = ['res' => false, 'msg' => "Please try again!", 'data' => ''];
+            }
+        }
+
+        return response()->json($response);
+    }
+
+
+    public function orders(Request $request)
```

Comment:
phpdocs missing, and move orders function into the sales/order module, and then create a sale/order object to call this function